test/conformance/testdata/conformance.yaml

```diff
@@ -42,9 +42,8 @@
   codename: '[k8s.io] Container Runtime blackbox test on terminated container should
     report termination message [LinuxOnly] as empty when pod succeeds and TerminationMessagePolicy
     FallbackToLogsOnError is set [NodeConformance] [Conformance]'
-  description: 'Name: Container Runtime, TerminationMessage, from log output of succeeding
```

Comment:
This change seems fine according to the corresponding e2e test code as https://github.com/kubernetes/kubernetes/blob/f52cbea102bb6ba9fe951cbf81e7dd4b48f95d9e/test/e2e/common/runtime.go#L249
The old description contained `Name:` part also, then the description seemed odd. New one seems correct.

Reply:
I submitted a follow up PR for this https://github.com/kubernetes/kubernetes/pull/88923.

Some conformance headers had "Name" instead of "Testname" and were picked up as part of the description by the tooling.